inc/compat/themes/compat-theme-minimog.php

```diff
@@ -22,6 +22,16 @@ public function hooks() {
 		// Scripts
 		add_action( 'wp_enqueue_scripts', array( $this, 'deregister_woocommerce_scripts' ), 20 );
 		add_action( 'wp_enqueue_scripts', array( FluidCheckout_Enqueue::instance(), 'replace_woocommerce_scripts' ), 20 );
+
+		// Remove checkout payment info heading
+		add_filter( 'fc_content_section_class', array( $this, 'remove_checkout_payment_heading' ), 20 );
+	}
+
+	 /**
+	 * remove checkout payment info heading added by the theme
+	 */
```

Comment:
Comment formatting a little bit off. The first line is not indented correctly, and the comment should start with a capitalized letter and end with `.`.